src/documentation/markdown.rs

```diff
@@ -99,7 +131,8 @@ impl<'a> PlainOverlay<'a> {
                     }
                 }
                 Event::Text(s) => {
-                    if code_block {
+                    if code_block || skip_text {
+                        skip_text = false
```

Comment:
Are nested links ok? If so, would this cause issues here?

Reply:
can you write an example of a nested link? I can check it.

Reply:
I.e. `[link of [links](./a)](./b)` (not sure this is legal, I did not check the spec)

Reply:
no, it does not support nested links. There is also not option in the `pulldown-cmark` crate